src/platform/utilities/api/index.js

```diff
@@ -18,6 +72,13 @@ export function fetchAndUpdateSessionExpiration(...args) {
         if (sessionExpiration) {
           localStorage.setItem('sessionExpiration', sessionExpiration);
         }
+        // SSOe session is independent of vets-api, and must be kept alive for continuity
+        const sessionExpirationSSO = localStorage.getItem(
+          'sessionExpirationSSO',
+        );
+        if (sessionExpirationSSO) {
```

Comment:
Does there need to be anything more to this condition e.g. that the sessionExpiration from localStorage is in the future?

Reply:
there's time-sensitive logic encapsulated in the `refreshSSOeSession` function, which could probably contain anything extra along those lines. 

that initial flag is more just a check for existence of any SSO session. Once logout logic gets explicitly added to the front-end, there'll need to be some added teardown of that item, as well.

Reply:
@hinzed1127 when the user first loads this JS, `localStorage.getItem('sessionExpirationSSO')` will default to `null` right?  If so how does the `refreshSSOeSession` get invoked for the first time?

Reply:
that's a good callout. I renamed the helper function to `checkAndUpdateSSOeSession` and absorbed that logic into there